### ISSUE TITLE
lint fails with no error

### ISSUE THREAD

**Issue by benmonro** [nrwl/nx#4198]
## Current Behavior

in CI, lint fails with no output, errors, info as to why.

## Expected Behavior

failures show some sort of error log.

## Steps to Reproduce

I wish i could, we have not yet been able to even repro this ourselves, just posting this in the hopes that maybe someone else has seen the issue.

This issue may not be prioritized if details are not provided to help us reproduce the issue.

### Failure Logs

```
20:16:00 + sh -c 'cd /opt/app && yarn nx lint home-tipping-modal'
20:16:00 yarn run v1.22.4
20:16:00 $ node --max-old-space-size=8192 ./node_modules/.bin/nx lint home-tipping-modal
20:16:11 
20:16:11 > nx run home-tipping-modal:lint 
20:16:13 
20:16:13 Linting "home-tipping-modal"...
20:17:39 error Command failed with exit code 1.
20:17:39 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
### Environment

Jenkins, docker (alpine)

**Comment by FrozenPandaz** [comment]
Sorry, there is not enough information here for us to take a look.

Some questions I have:
1. Do you see an error locally?
2. Are you using ESLint or TSLint?

If possible, can you please provide a repo that we can pull down to help us investigate the issue?

**Comment by cguilhermef** [comment]
@benmonro verify if in your root .eslintrc file you have this guys:

```json
{
      "files": ["*.ts", "*.tsx"],
      "extends": ["plugin:@nrwl/nx/typescript"],
      "rules": {}
    },
    {
      "files": ["*.js", "*.jsx"],
      "extends": ["plugin:@nrwl/nx/javascript"],
      "rules": {}
    }
```
If you ran a nx migration, in an old workspace, maybe that was not updated. I found some issues like yours - lint failing without prompt errors. I put this two objects in the overrides array and the errors appeared.

I found this diff comparing my repository with a fresh one that a created to compare.
